lib/ansible/modules/cloud/ovirt/ovirt_vms.py

```diff
@@ -47,6 +47,43 @@
         description:
             - Boolean indication whether to allow partial registration of Virtual Machine when C(state) is registered.
         version_added: "2.4"
+    vnic_profile_mappings:
+        description:
+            - "Mapper which maps an external virtual NIC profile to one that exists in the engine when C(state) is registered."
+        version_added: "2.5"
+    cluster_mappings:
+        description:
+            - "Mapper which maps cluster name between VM's OVF and the destination cluster this VM should be registered to,
+               relevant when C(state) is registered."
+        version_added: "2.5"
+    role_mappings:
+        description:
+            - "Mapper which maps role name between VM's OVF and the destination role this VM should be registered to,
+               relevant when C(state) is registered."
+        version_added: "2.5"
+    domain_mappings:
+        description:
+            - "Mapper which maps domain name between VM's OVF and the destination domain this VM should be registered to,
```

Comment:
Actually from this description I have no idea what `domain` means? storage/AAA domain or any other domain?

Reply:
changed to be aaa domain